global-config.js

```diff
@@ -10,4 +12,10 @@ exports.restApiRoot = '/api';
 // The URL where the browser client can access the REST API is available
 // Replace with a full url (including hostname) if your client is being
 // served from a different server than your REST API.
-exports.restApiUrl = exports.restApiRoot;
+exports.restApiUrl = url.format({
+  protocol: 'http',
+  slashes: true,
+  hostname: serverConfig.host,
+  port: serverConfig.port,
+  pathname: 'api'
+});
```

Comment:
Sorry if I have mislead you. The server config depends on global-config (see [server/config.local.js](https://github.com/strongloop/loopback-example-offline-sync/blob/master/server/config.local.js)), not the other way around.

If we need host and port config, then it should be set here in global-config and then used wherever needed.